packages/block-library/src/file/index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Server-side rendering of the `core/file` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * When the `core/file` block is rendering, check if we need to enqueue the `'wp-block-library-file` script.
+ *
+ * @param array $attributes The block attributes.
+ * @param array $content    The block content.
+ *
+ * @return string Returns the block content.
+ */
+function render_block_core_file( $attributes, $content ) {
+	if ( ! empty( $attributes['displayPreview'] ) ) {
+		// Check if it's already enqueued, so we don't add the inline script multiple times.
+		if ( ! in_array( 'wp-block-library-file', wp_scripts()->queue, true ) ) {
```

Comment:
```suggestion
		if ( ! wp_script_is( 'wp-block-library-file' ) ) {
```

Reference: https://developer.wordpress.org/reference/functions/wp_script_is/